uspace/lib/c/generic/rtld/symbol.c

```diff
@@ -64,6 +64,11 @@ static elf_word elf_hash(const unsigned char *name)
 
 static elf_symbol_t *def_find_in_module(const char *name, module_t *m)
 {
+	if (m->dyn.hash == NULL) {
```

Comment:
Technically should be a separate PR, but eh, whatever. Curious, did you happen to encounter a crash here?

Reply:
yep, with Rust programs. This was in the beginnings of the project, I don't really remember. I later had to configure the compiler to create the standard hash section, not the .gnu.hash extension. So maybe the binary that crashed was invalid anyways. Should I try to reconstruct it?

Reply:
Nah, you don't have to. More robust code never hurts.